csrc/python_frontend/fusion_definition.cpp

```diff
@@ -624,4 +626,12 @@ std::vector<std::pair<double, double>> FusionDefinition::getValTolerances(
   return get_val_constants(preschedFusion(), inputs);
 }
 
+void FusionDefinition::clone(FusionDefinition& other) {
+  NVF_CHECK(id().has_value(), "FusionDefinition definition does not exist!");
+  NVF_ERROR(
+      !other.completed(),
+      "Expected an incomplete definition before translation.");
+  translate(preschedFusion(), &other);
```

Comment:
I'm now questioning why `FusionDefinition::preschedFusion()` is a private method. That definition is just calling all public methods.
I'm leaning towards having this debug function by itself.